api/clients/v2/examples/example_relay_retrieval_test.go

```diff
@@ -0,0 +1,86 @@
+package examples
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/status-im/keycard-go/hexutils"
+)
+
+// This example demonstrates how to use the RelayPayloadRetriever to retrieve a payload from EigenDA, running on
+// holesky testnet
```

Comment:
This is exact same comment as the one for ValidatorRetriever. Maybe add a sentence or two describing the difference between the two, mention that this one is centralized retrieval, and point to the other one for retrieving from the eigenda nodes directly

Reply:
[e83b570e](https://github.com/Layr-Labs/eigenda/pull/1454/commits/e83b570ed6b8c20aebbc4b9f87bcfea8f23a4596)